chronix2grid/generation/consumption/consumption_utils.py

```diff
@@ -24,7 +24,7 @@ def compute_loads(loads_charac, temperature_noise, params, load_weekly_pattern,
     first_dow_chronics = datetime.strptime(load_weekly_pattern["datetime"].iloc[1], "%Y-%m-%d %H:%M:%S").weekday()
     # + (calendar.isleap(start_day.year) if start_day.month >= 3 else 0)
     day_lag = (first_dow_chronics - start_day_of_week) % 7
-    day_lag = 6  # this is only TRUE if you simulate 2050 !!!
+    day_lag = 0#6  # this is only TRUE if you simulate 2050 !!!
```

Comment:
Would it be possible to have this as argument of this function, with a possible default value of 0 if you'd like.
Even better would be to have it in a config file somewhere.

We really need it for wcci 2022

Reply:
I agree we should address this issue https://github.com/BDonnot/ChroniX2Grid/issues/45 and improvement. 

This should be developed through a next branch and PR, as this one was only to make sure that all tests were well-aligned without evolution in functionnalities + continuously run through CircleCI